src/Storages/Kafka/ReadBufferFromKafkaConsumer.cpp

```diff
@@ -82,18 +82,27 @@ ReadBufferFromKafkaConsumer::ReadBufferFromKafkaConsumer(
 
 ReadBufferFromKafkaConsumer::~ReadBufferFromKafkaConsumer()
 {
-    /// NOTE: see https://github.com/edenhill/librdkafka/issues/2077
     try
     {
         if (!consumer->get_subscription().empty())
             consumer->unsubscribe();
-        if (!assignment.empty())
-            consumer->unassign();
```

Comment:
this has been removed just because it is not required or it breaks something?

Reply:
https://github.com/confluentinc/confluent-kafka-go/issues/189#issuecomment-392047086